packages/terra-overlay/translations/de.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Reach out to Kelsey Emmons, she should be able to help verify the translations. I'll work on some docs for requesting translations in the future.